src/vercel.ts

```diff
@@ -88,7 +88,10 @@ const createAliases = async (deploymentUrl: string, customDeploymentFile: string
         .then((data) => data.json())
         .catch((error) => core.warning(`Did not receive JSON from Vercel API while creating aliases. Message: ${error?.message}`));
 
-      const aliasCreationPromises: Promise<VercelAliasResponse>[] = generateAliasPromises(id, ownerId, vercelConfig.alias);
+      const aliasAsked: string[] = [vercelConfig.alias, extraAliases].reduce((accumulator, value) => accumulator.concat(value), []).filter((alias) => alias ? true : false); // Merge both static and dynamic aliases, and make sure to remove any undefined element.
```

Comment:
`const aliasAsked: string[] =[vercelConfig.alias, extraAliases].reduce((accumulator, value) => accumulator.concat(value), [])`

can be simplified:
`const aliasAsked: string[] = [...vercelConfig.alias, ...extraAliases]`;